polysh/remote_dispatcher.py

```diff
@@ -73,18 +73,23 @@ def log(msg):
 class RemoteDispatcher(BufferedDispatcher):
     """A RemoteDispatcher is a ssh process we communicate with"""
 
-    def __init__(self, hostname):
+    def __init__(self, host):
```

Comment:
Again, you changed the type here from str to tuple. While at it I would probably not change it to tuple, but make this function take a hostname and a port argument separately. Then search for all places RemoteDispatcher is instantiated and make sure it's passed through properly everywhere.